README.md

```diff
@@ -39,3 +39,25 @@ module.exports = {
   }
 };
 ```
+
+### Precompile HTMLBars template strings within other addons
+
+```javascript
+module.exports = {
+  name: 'my-addon-name',
+
+  setupPreprocessorRegistry: function(type, registry) {
+    var htmlbarsPlugin = registry.load('template').find(function(plugin) {
+      return plugin.name === 'ember-cli-htmlbars';
+    });
```

Comment:
Yes, but we should totally add a `registry.findByName` that makes this a simple one liner.  Can be a future thing though.